GOVERNANCE.md

```diff
@@ -165,6 +174,84 @@ We have a reasonably liberal approach to code review and merging. We value quick
 - All code must go through Pull Requests (PR) and pass status checks before being merged. If a PR is merged that breaks `main` due to the branch not being up-to-date, then it should either be reverted or a quick fix merged as a separate PR.
 - If a PR is against code you have previously committed and is either small changes, bug fixes, or refactors, then you're free to merge it without any review. However, if you need more confidence in your changes, you can wait for approval from another maintainer or core contributor.
 
+## Financial Contributions
+
+Biome is in the fortunate position that people care enough about it that they want to invest financially in its future. However, with money comes responsibility, so we are committed to the following.
+
+There are two ways to fund Biome and its development: sponsorship and funded bounties, although we strongly prefer sponsorship.
+
+### Sponsorship
+
+Sponsorship is the first and foremost way of making financial contributions to Biome. Sponsorship can be
+done through a one-time donation, or through recurring donations. We offer incentives for certain levels of recurring donations.
+
+To learn more about sponsorship incentives, or to make a donation, please visit our [Open Collective page](https://opencollective.com/biome).
+You can also sponsor the project through [GitHub](https://github.com/sponsors/biomejs).
+
+### Community-Funded Bounties
+
+Bounties are the second approach towards funding Biome development. Unlike a sponsorship, bounties are directed towards the completion of a specific task. Tasks include, but are not limited to, the development of a feature or the fixing of a bug.
+
+Compared to sponsorship, bounties come with a few risks, both for Biome as a project, as well as the persons or entities offering and collecting the bounty. To minimize these risks, we only sanction bounties under the following conditions:
+
+- Bounties must be pledged through Polar.sh on an issue that is opened in our issue tracker.
+  - Unless otherwise specified, Biome reserves 30% of the payment amount, meaning contributors receive 70% of the pledged amount.
+  - The commission charged by Polar.sh is split accordingly.
+- If an issue in our issue tracker is not yet open for pledges, the person who wishes to pledge should indicate their desire in the comments of the issue, or do so through the `Community > #bounties` channel on Discord. A core contributor can then **accept** or **reject** the request for the bounty. Reasons for rejection include, but are not limited to: 
```

Comment:
All done, except the last point: I wonder if/hope it might be sufficient for now to let core contributors make these decisions at their own discretion, as long as it fits within the rest of the rules as stated in this document. Do we think a poll is necessary for each bounty request?